src/pages/dashboard/hooks/useDashboardData.ts

```diff
@@ -285,199 +231,23 @@ export const useDashboardData = (filters: any) => {
     filter: { isCover: true }
   });
 
-  const [page, setPage] = useState(1);
-  const PAGE_SIZE = 100;
-  const [allV3Projects, setAllV3Projects] = useState<any[]>([]);
-  const [isLoadingV3Projects, setIsLoadingV3Projects] = useState(false);
-  const [hasMoreV3Projects, setHasMoreV3Projects] = useState(true);
-  const [totalV3Projects, setTotalV3Projects] = useState(0);
-
-  const filterParams = useMemo(() => {
-    const params: HookFilters<typeof useProjectIndex> = {
-      status: "approved"
-    };
-
-    if (filters?.country?.country_slug?.trim() !== "") {
-      params.country = filters.country.country_slug;
-    }
+  const dashboardProjects = useMemo((): DashboardProjectsLightDto[] => {
+    if (!dashboardProjectsData || !Array.isArray(dashboardProjectsData) || dashboardProjectsData.length === 0)
```

Comment:
`Array.isArray()` will null / undefined check this for you, so the first boolean term is not needed.